tutorials/images/Images_Tutorial.ipynb

```diff
@@ -0,0 +1,779 @@
zwU0MchARKMLAuHx0rH2xuk6ulRSGRzpEJh80cwkHk+WQpUJa41LBlx1aOPH0Y7eU8YV+EGKMJwxCdGSpBEVsQlEolkIIsyTCBQllFIHPDjbyPckmbpgWRX8C5hMZok86pDtIJqmMNzJlF2v0uqTUEQUSmM6SDgTFIHOgSMT1k5lDSwyiTC9tlEWdT4igmMZaVwRqbjEP4ueTcWbBGoIRjMBBoBujYEfkBQlqM8hBBA1HM8FyRpqkwumGKbV3LY08+xU2vvgllQSFIUHg2zLM2PAGxwgsUUnVIzYBBdxGQrC72kDiM1jx/9BSry4uUiwWksnz7K9/jhgM38tBD38VTEalOkJ7CKgEyNyHJbIavJEmqGcQ9hB+AzdAWfGMwxoHnCFWA7wypdljdJzZ9nAEZ+AgrMJgfOh8KJwGDMfmRyDqLL8Ah0WkuupfCR2DRmUMqi1QR1jkyl4dphsLHmgTPdwycBO1wgMTDuTxY0heS1Dmsdpd+exkvVehWjO1ANFqh3dPg4MWnT/CDh5/lsp0bOXn8HMWKj+46cI4jp4+gLkhqO8epqBIXLyxz2TXriK3Gr0csrp3j1IlVgpEC93zvS3TTlPSk5pa33crC7Bx6PuFC5wI/9bobGHEBsl5jEK/hKOJnA1qrCeMNy+W7drGY9rC6QxgUmVq/gYcfeorVwRoH9u5i4Pscjy8SVEp8/9QZxmYVuryOL33tPl7/ttezdf0WTh47zneffILUaEQfOmstrr/2GrpzJ5mYGIcspT4xyg37DzCblRj3e8Rdye71dRLnOJ90mL9wmqmpKaY7y/n9yixpAO3T0zx9fImJDTN84dvf4dbbb+PM/BxPHj7C/MJ5fu5n3o1Jc12tcGB13kB21v53E7kfevBB+v0uzgl6/T6FKKJWr1OpVJDCY8+ePWgcS+fP8od//ml27N5FYhydpXneeP1ejhw7xO4rd1ItlVhdWyYqVBBhzE1XXcvUFXtZFYYPvf+91Me38bG/+yqvuWkvszMv8oW//wbzsws8/vzTjE2MUh0fQxXrLK+uUvYhHbQpFEpUKiXmR+uEKqIclmhUq2wca7J4+hj++AgihR0lx613XIsjJZQeDoOdLIN2rFw8S7taYFYEyDTDmAwlDFfs2cXg6GFGPENYrDBxxTC3bN3E6RfP4BnDWtxFmICJqfV86hMfZ7hSQ2zcyIXVDn67i+h0iaoVgtV54jjmxqkS7dYS60aHKRYU00ee49V3vAJpYoZLAXfctJ/Uxdx/z3doVkLefdsBfAHtOGb62R/A3ArCwO6r91CqF2hMTDK+rsZC4pida7NpvMvMxTmGKzWWWh2El6G1wgs8Cv2ENpZWr4uq1H/kffqSKEaSvkWEEp05lAxxSYZxBnSCc47FxQ7SL1ATjhuGq7B+Atlfw1MRfZ1SGRvD37iJRTLKTpD0+vjVArrdxsssFyOJLNSwz/2AYHI9slLHlArY1CAKCun5FAs1ZLGCxRFc0oustXr0TIx1lvbFWaIoYsnCyrnzfOyvPonv+ww1h2k0hhDJgObWfWzZvgkPwR997D/zlre8haJ09PwO/8+37ufhb3ydQrnAZVfsohBGzC0uUMUgE8fPblJct+sK7r3/UYYbVXzlEMIj03nC9PziAtdceQC/GDBUlRgNU5vrLEyfZXTjNtb0EoqMxe9OI7yMUrVImGUMAovnQ92PcK6AVAHFYg1rNan28cIaqdEkwlEbHkaqBXpzc1gh6ffW6AUR6xrDdHoDVlPLubWM4eGRH/eSeRk/gXACfHLVcAGPtosJnSR14DlFMRXEfkrgPFJncUrjNAwCj4IBTO40lbqMSHmYrsGojHJ9mDjpU/OK9JVBk2KsoGs1vjAIK3BOkFqLZwXOzw0cFIIs1Rgc1jqEJ/EvJcKnlxy3JsabLM7NgnVMjIyyenoOVSiQJgm+UyRZShDmAlCtNXHSweoMoXyEdShPYRKDthqlJMovMlQpstJuI1BYa5C+xPVTQl/klFaliGxEQkaKJXAu74YJh8XLM/p8D4iR1SpKeEijUCi08MiyJYxXI7SORMcoFdIbtJGBj681fZGhEvB8R6IdcdxjYWYWz4fmRJPFuRVGNk7y4rOH2LxnG2dPTJPGCcUoIiOnnyZxjBQGazRJPMD3ChgdI6QEp3KPfU+BcAz6Mf20i5TgtMMJhxd4xFqjjAMULi898C/x1Q3gnMGJPFAtEAJHXkhorVEShMztmJ0zWGPAgPIcSToAfKS2eFZiPYUxFmMNOHLKlpWARgiBUi9b+76UEdQ9zKolXu4jfI/RXZPUyooTjw84Zc8xvGGc1toc/agLRbhm6zUcWjlI69QCAXXe+rbb+bPf/wKbbyty5myf8ZH97Nwdcujbsyy2UsYuX0dSmaPeVAzXbqWj5jn2yHPs//AGZg5nrLUzQiXJvA5ZPM/xkzOMDNcYWWfoDPpkWnNybpZqMMz58xcY3z6FtR5ZLFhOEkRPkxpNMRpClEOalVEq5SYDFXNmeY1r917BVddeyeP3/gCvPMSurdv54n/7Pt3WMjrNqA8NM7llE5/962d41zuvZjA3y4nleUwvobZulHe87k4+/8WvMjU1icQhpCP1HYPAsXm8yMX5FZqNMlG1Stm0ePZomze+7W0cPXmKqeYIzl1yYTKWUqEI/wRF6x+cUJ1zhJ7KCwHlYZ3CWY24NGlUwsup+jJ/lzaaNUqNKo1ihaTTI1QegedR8gts2LSR6mpENkg5d/oC+w9cw5/+1SfpmDKX7b0DJWFsdD3nlyQ/9xu/T295ml/54Lup+Dfx4Q99BKnguqt2s3NqgmTmLIO0w81jYwzVStw+ei1KCVIrSLIMF1SYuXAUEYXoIGL7NddiXJ6Dp3UKSlKu1HJ735kXuGLnlUiR4DxodfpEUZlC4Ofvn0aBEyemqe/ZSLM+wgl7jsxmIPPvxvhok7uu3YtfLtCPEzYXBEttTZYZOms9CvUmy61l0qESf/+ZP0P4ikIU5fe1Wsf3Fb
```

Comment:
I would just show learned & empirical accuracy- and be clear the latter is just over the dev set!